Automattic/pocket-casts-ios
# Categories: Animation Tweaks

Fixes #1647

* Cleans up the pills animations by changing the selected pills to be outside of a scroll view, this prevents the prior scrolling behavior.
* Fixes an issue with the placeholders pills not matching the position of the loaded pills

https://github.com/Automattic/pocket-casts-ios/assets/3250/befc9aa7-b13a-4642-9657-70c96b6cb102

## To test

* Navigate to Discover
* Scroll the Category pills all the way over to the right
* Tap a pill
* Notice that the animation should be quicker and look a bit smoother.

## Checklist
